### PR TITLE
Make code Python3 compatible and perform cosmetic changes

### PR DESCRIPTION
**Pull request by Vages**
The step counting parts of your repo could turn out to be very useful for my master’s thesis. Therefore I have taken the time to make it Python3 compatible, as well as making some aesthetic changes (PEP-8, deleting unused imports and variables).

### CODE REVIEW

**Comment by danielmurray** [comment]
LGTM, can you point out specifically which parts of the code are not Python3 compatible, just for my knowledge.

**Comment by Vages** [comment]
@danielmurray Will do (not for every print statement to function conversion, though). Outside of that, there’s just one place, I think.

File: sensorData/result.py [hunk 1]
```diff
@@ -1,122 +1,122 @@
...
-			"inHand": 100,
-			"inPocket": 100
-		},
-		'walk': {
-			"inHand": 100,
-			"inPocket": 100
-		}
-	}
-
-	for test, sub_tests in tests.iteritems():
```
Comment (Vages) [inline sensorData/result.py:70]: Not Python3-compatible

File: sensorData/result.py [hunk 2]
```diff
@@ -1,122 +1,122 @@
...
+            "inHand": 100,
+            "inPocket": 100
+        },
+        'walk': {
+            "inHand": 100,
+            "inPocket": 100
+        }
+    }
+
+    for test, sub_tests in iter(tests.items()):
```
Comment (Vages) [inline sensorData/result.py:71]: Updated to be Python3 (and 2) compatible as per http://legacy.python.org/dev/peps/pep-0469/

**Review by Vages — COMMENTED** [review]
The only Python3-incompatible place (except for all print statement-to-function updates).

File: .gitignore [hunk 3]
```diff
@@ -4,3 +4,140 @@
 /.idea/libraries
 .DS_Store
 /build
+
```
Comment (danielmurray) [inline .gitignore:7]: Why are these files in your root project folder?

**Review by danielmurray — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 3 above)
Comment (Vages) [reply .gitignore:7]: Are you thinking of the lines I have added? Those are taken from Github’s official Python .gitignore (https://github.com/github/gitignore/blob/master/Python.gitignore) and Jetbrains .gitignore files (https://github.com/github/gitignore/blob/master/Global/JetBrains.gitignore).

Comment (danielmurray) [reply .gitignore:7]: I'm alright with the python standard .gitignore, but let's take out the Jetbrains .gitignores

Comment (Vages) [reply .gitignore:7]: That should do it.

**Comment by danielmurray** [comment]
LGTM

**Pull request merged**